package.json

```diff
@@ -9,6 +9,7 @@
     "esbuild": "^0.21.5",
     "govuk-frontend": "5.4.0",
     "jquery": "^3.7.1",
+    "puppeteer": "^22.12.0",
```

Comment:
is this being out of sync with the lib version a risk?

```
"puppeteer": "^19.2.0"
```

Reply:
Have removed the lib version